transformato/schrodinger_systems.py

```diff
@@ -7,7 +7,144 @@
 from transformato.utils import load_config_yaml
 from transformato.constants import check_platform
 
+def cdk2_type1(conf_path: str, input_dir: str, output_dir: str):
+    configuration = load_config_yaml(
+        config=conf_path, input_dir=input_dir, output_dir=output_dir
+    )
+
+    s1 = SystemStructure(configuration, "structure1")
+    s2 = SystemStructure(configuration, "structure2")
+    s1_to_s2 = ProposeMutationRoute(s1, s2)
+    #s1_to_s2.calculate_common_core()
+    s1_to_s2.propose_common_core()
+    connected_dummy_regions_cc1=[set(s1_to_s2.get_idx_not_in_common_core_for_mol1())]
+    connected_dummy_regions_cc2=[set(s1_to_s2.get_idx_not_in_common_core_for_mol2())]
+
+    s1_to_s2.finish_common_core(connected_dummy_regions_cc1=connected_dummy_regions_cc1, connected_dummy_regions_cc2=connected_dummy_regions_cc2)
+
+    ###############################
+    ####### Structure 1    ########
+    ###############################
+    
+    # generate the mutation list for the original
+    mutation_list = s1_to_s2.generate_mutations_to_common_core_for_mol1()
+    print(mutation_list.keys())
+    output_files = []
+    i = IntermediateStateFactory(
+    system=s1,
+    configuration=configuration,
+    )
+
+    # write endpoint mutation
+    output_file_base, intst = i.write_state(mutation_conf=[], intst_nr=1)
+    output_files.append(output_file_base)
+
+    # start with charges
+    # turn off charges
+    charges = mutation_list["charge"]
+    nr_of_mutation_steps_charge = (
```

Comment:
can you also put the number of mutation steps for the different mutations in the yaml file?

Reply:
Good idea! I added them for each structure...